<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-FS5UPGRD" failures="5" tests="5" name="masterTest(failed)" time="58.356" errors="0" timestamp="2023-03-19T16:26:35 IST">
  <testcase classname="testcases.TC_002Logintest" name="test_login" time="32.302">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at testcases.TC_002Logintest.test_login(TC_002Logintest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase classname="testcases.TC_002Logintest" name="test_login" time="4.165">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at testcases.TC_002Logintest.test_login(TC_002Logintest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase classname="testcases.TC_002Logintest" name="test_login" time="0.067">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at testcases.TC_002Logintest.test_login(TC_002Logintest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase classname="testcases.TC_002Logintest" name="test_login" time="0.049">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at testcases.TC_002Logintest.test_login(TC_002Logintest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase classname="testcases.TC_002Logintest" name="test_login" time="0.047">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at testcases.TC_002Logintest.test_login(TC_002Logintest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
</testsuite> <!-- masterTest(failed) -->
